cluster/gce/upgrade-aliases.sh

```diff
@@ -50,11 +51,12 @@ function get-k8s-node-routes-count() {
 # Vars set:
 #  IP_ALIAS_SUBNETWORK
 function detect-k8s-subnetwork() {
-  local subnetwork_url=$(gcloud compute instances describe \
-    ${KUBE_MASTER} --project=${PROJECT} --zone=${ZONE} \
+  subnetwork_url=$(gcloud compute instances describe \
+    "${KUBE_MASTER}" --project="${PROJECT}" --zone="${ZONE}" \
     --format='value(networkInterfaces[0].subnetwork)')
+  local subnetwork_url
   if [[ -n ${subnetwork_url} ]]; then
-    IP_ALIAS_SUBNETWORK=$(echo ${subnetwork_url##*/})
+    IP_ALIAS_SUBNETWORK="${subnetwork_url##*/}"
```

Comment:
why change: 
```
In ./cluster/gce/upgrade-aliases.sh line 57:
    IP_ALIAS_SUBNETWORK=$(echo ${subnetwork_url##*/})
                        ^---------------------------^ SC2116: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'.
                               ^-------------------^ SC2086: Double quote to prevent globbing and word splitting.
```
Ref: https://github.com/koalaman/shellcheck/wiki/SC2116